solanaj/src/main/java/org/p2p/solanaj/ws/SubscriptionWebSocketClient.kt

```diff
@@ -156,7 +155,11 @@ class SubscriptionWebSocketClient(serverURI: URI?) : WebSocketClient(serverURI)
 
         companion object {
             fun valueOf(type: String): NotificationType? {
-                return values().find { it.type == type }
+                return values().find { it.type == type }.also {
+                    if (it == null) {
+                        Log.e("NotificationType", "Unknown NotificationType: $type")
```

Comment:
it's better to create and pass an instance of some Throwable here, we need to create an entry in Crashlytics for such a case. 
Right now it's just logging this error into a console.